packages/sitecore-jss-react/src/components/Link.tsx

```diff
@@ -135,6 +140,17 @@ export const LinkPropTypes = {
       value: PropTypes.object,
       editableFirstPart: PropTypes.string,
       editableLastPart: PropTypes.string,
+      metadata: PropTypes.shape({
```

Comment:
I think it makes sense to create a shared _PropTypes_ that defines _metadata_ proptype and reuse it across all the fields